package.json

```diff
@@ -19,8 +32,8 @@
   "scripts": {
     "build": "run-s build:*",
     "build:0-clean": "rimraf dist coverage",
-    "build:1-compile": "tsc -p tsconfig.build.json",
-    "build:2-copy-pkg-json": "npx --yes @makerx/ts-toolkit@latest copy-package-json",
+    "build:1-compile": "rollup -c --configPlugin typescript",
```

Comment:
I'm assuming that Rollup will pick up on the `rollup.config.ts` config file automatically, which already defines the typescript plugin. Do we need to specify it here explicitly?

Reply:
Exactly right, `rollup.config.ts` is a default that Rollup finds, like if we use just `tsc` it finds `tsconfig.json`. We could explicitly supply the arg though?

Reply:
I'm happy to not be explicit and lean on the convention.

What I was mentioning is that we seem to specify the typescript plugin in both the config file and this `rollup` command.

Do we need both?

Reply:
Ah right, now I understand. The `--configPlugin typescript` allows Rollup to read the Typescript config file. Without that, it can't even read the config, it would have to be written in JS.